README.md

```diff
@@ -1 +1,33 @@
-# Awsome-Books-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awesome Books
+
+**What I did in this project:**
+
+1.  Add book class with constructor.
+2.  Include add and remove methods in the class.
+3.  Make these methods to add and remove books data to DOM and local storage.
+4.  Add some basic styling and check linters errors.
+
+# Site Screenshots
+
+![screenshot](Website-screenshoot.png)
+
+## Built With
+
+- HTML.
+- CSS.
+- JavaScript.
+- Linters.
+
```

Comment:
- [ ] I would recommend mentioning the steps on the README on how to start the project. It's very important to include the steps if the user/recruiter wants to run the project locally but doesn't get proper steps. It gives a bad impression to the recruiter about you. Please update that.